.gitignore

```diff
@@ -160,3 +160,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+results/
+**/miniconda.exe
```

Comment:
What this line do?
If you use conda please put the conda env folder in another folder (not in the repository folder)